### PR TITLE
allow volunteer to change pickup and delivery instructions

### PR DESCRIPTION
**Pull request by mat10tng**

### CODE REVIEW

File: src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx [hunk 1]
```diff
@@ -77,11 +141,17 @@ const MissionDetailsCard = ({ mission, photoDisabled }) => {
...
   const classes = useStyles();
 
-  if (mission.status === "started") {
+  if (mission.status === Mission.Status.started) {
     photoDisabled = false;
   } else {
     photoDisabled = true;
   }
+
+  let notesEditDisabled = false;
```
Comment (qdozaq) [inline src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx:150]: This should come from a `useState` call

Comment (qdozaq) [reply src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx:150]: err.. Oh I see why you did this.

Could it instead be:

```js
const notesEditDisabled = [Mission.Status.unassigned, Mission.Status.tentative].includes(mission.status);
```

Makes it more clear we're only using once

File: src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx [hunk 2]
```diff
@@ -66,6 +70,66 @@ const Row = ({ children, Icon, label }) => {
...
+`;
+const TextRowEditable = ({ disabled, label, onClick, text }) => {
+  const [value, setValue] = useState(text);
+  const [isEdit, setIsEdit] = useState(false);
+
+  function handleChange(event) {
+    setValue(event.target.value);
+  }
+  //disabled have higher priority
+  let isEditable = isEdit;
```
Comment (qdozaq) [inline src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx:87]: Could we do something similar to how you handled line 95?

It's just odd to reassign variables like this especially when they're already tied to a state

File: src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx (hunk 1 above)
Comment (mat10tng) [reply src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx:150]: yeah, you are right! It should look like that

File: src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx (hunk 2 above)
Comment (mat10tng) [reply src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx:87]: yeah, you are right

File: src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx (hunk 1 above)
Comment (mat10tng) [reply src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx]: resolved

**Comment by mat10tng** [comment]
You guys can merge in accordingly

**Pull request merged**